src/mumble/UserModel.cpp

```diff
@@ -1365,6 +1365,25 @@ bool UserModel::dropMimeData(const QMimeData *md, Qt::DropAction, int row, int c
 
 	if (! isChannel) {
 		// User dropped somewhere
+		int ret;
+		switch (g.s.ceUserDrag) {
+			case Settings::Ask:
+				ret=QMessageBox::question(g.mw, QLatin1String("Mumble"), tr("Are you sure you want to drag this user?"), QMessageBox::Yes, QMessageBox::No);
+
+				if (ret == QMessageBox::No)
+					return false;
+				break;
+			case Settings::DoNothing:
+				g.l->log(Log::Information, MainWindow::tr("You have User Dragging set to \"Do Nothing\" so the user wasn't moved."));
+				return false;
+				break;
+			case Settings::Move:
+				break;
+			default:
```

Comment:
Please leave out the default case.

The compiler should be smart enough to warn if we don't handle all cases, if we add new ones.

(I know the default case is there for channel drags, but there is no need to add the extra translation string here...)